temba/links/models.py

```diff
@@ -218,6 +218,12 @@ def update_node(node):
                 revisions.append(revision)
         FlowRevision.objects.bulk_update(revisions, ["definition"])
 
+    def get_clicks_count(self):
```

Comment:
I believe the `clicks_count` field may be deleted, as it was replaced by the `get_clicks_count ` method.
https://github.com/greatnonprofits-nfp/rapidpro/blob/994ac5446775fe25dd9ca5cc88d380c6e2146fc2/temba/links/models.py#L53

Reply:
Got it. Actually, I kept this just in case if we would like to go back using the original approach and having the original count saved, but I agree that makes sense removing this field. Thanks.